src/main/java/com/ait/lienzo/client/core/shape/wires/handlers/impl/AlignAndDistributeControlImpl.java

```diff
@@ -526,15 +521,15 @@ private void indexOn(AlignAndDistributeControl handler)
         if (handler != null && handler.isIndexed())
         {
             m_alignAndDistribute.indexOnWithoutChangingStatus(handler);
-            ((AlignAndDistributeControlImpl)handler).m_indexedButRemoved = true;
+            ((AlignAndDistributeControlImpl)handler).m_indexedButRemoved = false;
```

Comment:
hey @Josephblt , tbh a bit lost about this change, I don't know this code quite well... but why this change?